docs/guides/guide-vmanomaly-vmalert.md

```diff
@@ -0,0 +1,403 @@
+# VmAnomaly Quickstart
+
+### Prerequisites
+- In this tutorial it is assumed that you have VictoriaMetrics already, and you are familiar with VictoriaMetrics ecosystem:
+  -  [victoriametrics](https://docs.victoriametrics.com/Single-server-VictoriaMetrics.html)
+  -  [vmalert](https://docs.victoriametrics.com/vmalert.html)
+  -  [vmagent](https://docs.victoriametrics.com/vmagent.html)
+  
+- It is assumed that you are familiar with Grafana and Docker
+## What is VmAnomaly?
+*VictoriaMetrics Anomaly Detection* is a service that continuously scans VictoriaMetrics time series and detects unexpected changes within data patterns in real-time. It does so by utilizing user-configurable machine learning models.
+
+All the service parameters (model, schedule, input-output) are defined in a config file.
+
+Single config file supports only one model, but it's totally OK to run multiple vmanomaly processes in parallel, each using its own config.
+
+VmAnomaly periodically queries user-specified metrics, computes an **anomaly score** for them, based on how well they fit a predicted distribution, taking into account periodical data patterns with trends and/or seasonality, and pushes back the computed "anomaly score" to VictoriaMetrics. Then, users can enable alerting rules based on the "anomaly score" with VmAlert.
+
+
+## What is VmAlert?
+VMalert is an alerting tool for VictoriaMetrics. It executes a list of the given alerting or recording rules against configured `-datasource.url` compatible with Prometheus HTTP API.
```

Comment:
I know this sentence is copied, and this is why it contains information that user won't need in this guide.
For example, `compatible with Prometheus HTTP API` is not needed here. It is important to be focused only on things that are relevant to the current guide and give only important information, to keep it simple.

Reply:
This chapter is also missing a link to the vmalert docs for curious readers

Reply:
Changed, added link